exporter/kafkaexporter/jaeger_marshaler_curer.go

```diff
@@ -25,7 +25,8 @@ const (
 	maxTruncationTries           = 5      // maximum number of times to attempt to truncate tag values.
 	// suffix used for new attributes created for those whose values have been truncated
 	// while curing the spans
-	truncationTagSuffix = ".htcollector.truncated"
```

Comment:
Since `maxTruncationTries` is a constant, can we introduce 
```
maxNumberSpanLogs = math.Pow(2, maxTruncationTries) // float64
or 
maxNumberSpanLogs = int(math.Pow(2, maxTruncationTries)) // if you need int
```

Then we can prevent the calculation each time on line 207.  

Also we can remove lines 309-320.

Reply:
Apparently this is math.Pow() is slower than doing the multiplication loop for ints. https://stackoverflow.com/questions/64108933/how-to-use-math-pow-with-integers-in-golang

Reply:
Isn't it calculated only once?

Reply:
Yes for every span. And thousands of spans going through this per second.

Reply:
fixed as tyrone wanted. since the computation depends on  a const then we should only compute it once and put it in a var.